packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -885,15 +885,15 @@ curl -v -X GET \
 
 #### Delete a resource
 
-<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetId}/resources/${resourceId}" />
+<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetIdOrLabel}/resources/${resourceId}" />
 
 Removes a resource from a Resource Set
 
 ##### Request parameters
 
 | Parameter      | Description                             | Param Type   | DataType     | Required |
 | :------------- | :-------------------------------------- | :----------- | :----------- | :------- |
-| `resourceSetId`  | ID of the Resource Set                  | URL          | String       | TRUE     |
+| `resourceSetIdOrLabel`  | Unique ID or label of the Resource Set                  | URL          | String       | TRUE     |
```

Comment:
To keep it consistent, should this be just called `ID` instead of `Unique ID` ?

Reply:
Updated